ui/static/app/auth/auth.service.ts

```diff
@@ -58,6 +61,19 @@ export class AuthService {
   getUserId(): number {
     return Number(this.localStorage.get("auth_user_id"));
   }
+
+  isAdmin(): boolean {
+    return (this.localStorage.get("is_admin") === "true") ? true : false;
```

Comment:
Yes, is ok, n localStorage is saved as a string "true" and method is admin must return a boolean.